src/transformers/models/mamba2/modeling_mamba2.py

```diff
@@ -153,6 +163,11 @@ def __init__(
         self.activation = config.hidden_act
         self.act = ACT2FN[config.hidden_act]
 
+        # FIXM:E 
```

Comment:
will need to handle the `max_seq_length` hardcodes, and also understand what `lengths_per_sample` is

Reply:
not doing. abandoning `modeling_mamba2`